static/src/javascripts/projects/common/modules/onward/breaking-news.spec.js

```diff
@@ -17,13 +17,29 @@ jest.mock('lib/storage', () => ({
         isAvailable: jest.fn().mockReturnValue(true),
     },
 }));
-jest.mock('lib/config', () => ({
-    page: {
-        pageId: '12345',
-        edition: 'UK',
-        section: 'football',
-    },
-}));
+/**
+ * we have to mock config like this because
+ * loading simple-reach has side affects
+ * that are dependent on config.
+ * */
```

Comment:
Comment may want updating; this is the breaking-news Spec 😉

Reply:
hah, yep that was a copy and paste mistake, good spot